DataFormats/simulation/include/SimulationDataFormat/MCCompLabel.h

```diff
@@ -0,0 +1,97 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
@sawenzel : I added extension of the digits/clusters/tracks MClabels by the event and MCsource ID, but to minimize future modification, instead of simply using Long64 manipulated locally, I introduced base class for such a composed labels. Would you mind to check if this is compatible with your planned scheme for separate labels stream? I guess in any case it should store an objects of this kind.